Sources/Hypertext.swift

```diff
@@ -6,18 +6,23 @@
 //  Copyright © 2016 Sahand Nayebaziz. All rights reserved.
 //
 
-public protocol Renderable {
-    func render() -> String
-    func render(startingWithSpacesCount: Int) -> String
+public protocol Renderable: CustomStringConvertible {
+    func render(indent: Int?, startingWithSpacesCount: Int) -> String
```

Comment:
I think so. I'm going to load this version of Hypertext into a new package and see what it feels like with this naming.